modules/microshift-lvms-configuring.adoc

```diff
@@ -36,12 +36,17 @@ device-classes: <2>
 <2> `map[string]DeviceClass`. The `device-class` settings. 
 <3> String. The name of the `device-class`.
 <4> String. The group where the `device-class` creates the logical volumes. 
-<5> unit64. Storage capacity in GiB to be spared. Defaults to `10`. 
+<5> unit64. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `10`. 
 <6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`. 
 <7> unit. The number of stripes in the logical volume.
 <8> String. The amount of data that is written to one device before moving to the next device. 
 <9> String. Extra arguments to pas `lvcreate`, for example, `[--type=raid1"`]. 
 +
+[WARNING]
+====
+There is a race condition that prevents LVMS from accurately tracking the allocated space and preserving the `spare-gb` for a device class when multiple PVCs are created simultaneously. Use separate volume groups and device classes to protect the storage of highly dynamic workloads from each other reliably.
```

Comment:
Suggestion: 
There is a race condition that prevents LVMS from accurately tracking the allocated space and preserving the `spare-gb` for a device class when multiple PVCs are simultaneously created. Use separate volume groups and device classes to reliably protect the storage of highly dynamic workloads from each other.

(move the words "simultaneously" and "reliably")

Reply:
The word "simultaneously" is significant. There is no race condition if you make a PVC, wait for the PV, then make another PVC.

Maybe "concurrently" is a better choice?

Reply:
I did remove the "reliably"